### PR TITLE
Fix Fenced Doc-String #782

### PR DESCRIPTION
**Pull request by atharvapurdue**
Following changes were made in Doc-String Examples -  

- Add models to the list in docstring_test.py
- Import from the root of the library
- Fix typo in f_net_classifier.py
- add [MASK] in Vocaburay
- remove num_heads parameter 

The number of failed examples has gone down from 5 to 1, however I am getting 1 error in `File "/content/keras-nlp/keras-nlp/keras-nlp/keras_nlp/models/f_net/f_net_masked_lm.py", line 73, in keras_nlp.models.f_net.f_net_masked_lm.FNetMaskedLM`- 

`Node: 'f_net_masked_lm_1/f_net_backbone_3/segment_embedding/embedding_lookup'
    indices[0,3] = 4 is not in [0, 4)
    	 [[{{node f_net_masked_lm_1/f_net_backbone_3/segment_embedding/embedding_lookup}}]] [Op:__inference_train_function_458156]
`
Might need help from @abheesht17 , or any other developer who can assist.
Thanks!
Fixes keras-team/keras-nlp #782

### CODE REVIEW

**Comment by abheesht17** [comment]
Thanks, @atharvapurdue! The reason why it's throwing an error is because the default value of  `num_segments` is 4 for FNet. Hence, this throws an embedding lookup error: https://github.com/keras-team/keras-nlp/blob/master/keras_nlp/models/f_net/f_net_backbone.py#L126-L131. The best way to resolve it is to just change

```
        "segment_ids": tf.constant(
            [[1, 0, 0, 4, 0, 6, 7, 8]] * 2, shape=(2, 8)
        ),
```

to values less than 4. Just change it to something like  `[[0, 0, 0, 1, 1, 1, 0, 0]] * 2`.

**Comment by atharvapurdue** [comment]
Yes! @abheesht17. I thought so, but wasn't sure. I will get make the necessary changes. Thank you!

**Comment by atharvapurdue** [comment]
Done !  @abheesht17 , Thanks for your help!
You can see the doc_string_test passing all tests here - https://colab.research.google.com/drive/1xVlUGJIDSH44bF851HZ01yD24ocJmul5#scrollTo=8DErjytmUTv_

File: keras_nlp/models/distil_bert/distil_bert_preprocessor.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ class DistilBertPreprocessor(Preprocessor):
 
     # Alternatively, you can create a preprocessor from your own vocabulary.
     # The usage is exactly the same as above.
-    vocab = ["[PAD]", "[UNK]", "[CLS]", "[SEP]"]
+    vocab = ["[PAD]", "[UNK]", "[CLS]", "[SEP]","[MASK]"]
```
Comment (mattdangerw) [inline keras_nlp/models/distil_bert/distil_bert_preprocessor.py:142]: space after last comma

File: keras_nlp/tests/doc_tests/docstring_test.py [hunk 2]
```diff
@@ -114,6 +114,9 @@ def test_fenced_docstrings():
             "keras_nlp.models.xlm_roberta.xlm_roberta_preprocessor",
             "keras_nlp.models.f_net.f_net_preprocessor",
             "keras_nlp.models.f_net.f_net_tokenizer",
+            "keras_nlp.tokenizers.byte_pair_tokenizer",
```
Comment (mattdangerw) [inline keras_nlp/tests/doc_tests/docstring_test.py:117]: I think these three additions should be in the "base classes" block a few lines up.

**Review by mattdangerw — APPROVED** [review]
Looks good, a few minor comments.

File: keras_nlp/tests/doc_tests/docstring_test.py (hunk 2 above)
Comment (atharvapurdue) [reply keras_nlp/tests/doc_tests/docstring_test.py:117]: Made the required changes @mattdangerw. Please have a look

**Comment by mattdangerw** [comment]
Thank you! Works great!

**Pull request merged**